### DISCUSSION TITLE
提供客户端（Android && iOS）应用程序

### DISCUSSION THREAD

**Discussion by GeekTR** [AlistGo/alist#4446] [Show and tell]
### Please make sure of the following things

- [X] I have read the [documentation](https://alist.nn.ci).
- [X] I'm sure there are no duplicate issues or discussions.
- [X] I'm sure this feature is not implemented.
- [X] I'm sure it's a reasonable and popular requirement.

### Description of the feature / 需求描述

我们使用 Flutter 开发了基于 alist 后端 api 的客户端 App，支持 Android 和 iOS平台，代码开源链接如下：https://github.com/BFWXKJGS/AlistClient  。我们希望可以把App上架到 Apple 的 AppStore，方便大家的下载和使用，我们可以这么做吗？希望得到你们的授权，谢谢。

**Comment by welcome** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by xhofe** [comment]
当然！很欢迎你们的贡献，如果可以的话，欢迎将仓库转移到alist-org的组织下。

**Comment by anwen-anyi** [comment]
有打包好的 ipa么 试试看 效果 :kissing_heart:

**Reply by GeekTR** [reply]
我这边给的 ipa 你那边可以运行吗？我这边是个人的 apple 账号，提供不了可直接安装的ipa。如果你有办法可以运行的话，可以留个邮箱

**Reply by anwen-anyi** [reply]
> 我这边给的 ipa 你那边可以运行吗？我这边是个人的 apple 账号，提供不了可直接安装的ipa。如果你有办法可以运行的话，可以留个邮箱

可以 我用 TrollStore 同时也越狱了 
anwen_anyi@foxmail.com

**Reply by GeekTR** [reply]
好的，我这边下班后打个包发你邮箱

**Reply by xiaowang7212** [reply]
> 好的，我这边下班后打个包发你邮箱

也给我提供一份ipa吧，我这边有个人开发者证书签名。

**Reply by xiaowang7212** [reply]
517171882@qq.com谢谢

**Comment by SfYangYang** [comment]
希望尽快上架app store。在此之前，可以通过testflight让我们先用吗？

**Reply by anwen-anyi** [reply]
😆我知道你很急 但是请你先别急  哈哈

**Comment by GeekTR** [comment]
需要安装包的可以加qq群，143068783

**Comment by GeekTR** [comment]
https://apps.apple.com/cn/app/alclient/id6449257429
iOS版本已上架 App Store

**Reply by SEELE0** [reply]
可否上架谷歌商店

**Reply by GeekTR** [reply]
https://play.google.com/store/apps/details?id=com.github.alist.client
